enarx-keep-sev-shim/src/print.rs

```diff
@@ -113,3 +113,28 @@ macro_rules! eprintln {
     ($fmt:expr) => ($crate::eprint!(concat!($fmt, "\n")));
     ($fmt:expr, $($arg:tt)*) => ($crate::eprint!(concat!($fmt, "\n"), $($arg)*));
 }
+
+/// Prints and returns the value of a given expression for quick and dirty
+/// debugging.
+#[macro_export]
+macro_rules! dbg {
+    () => {
+        $crate::eprintln!("[{}:{}]", file!(), line!());
+    };
+    ($val:expr) => {
+        // Use of `match` here is intentional because it affects the lifetimes
+        // of temporaries - https://stackoverflow.com/a/48732525/1063961
+        match $val {
+            tmp => {
+                $crate::eprintln!("[{}:{}] {} = {:#?}",
+                    file!(), line!(), stringify!($val), &tmp);
+                tmp
+            }
+        }
+    };
+    // Trailing comma with single argument is ignored
+    ($val:expr,) => { $crate::dbg!($val) };
```

Comment:
```suggestion
    ($val:expr $(,)?) => {
        // Use of `match` here is intentional because it affects the lifetimes
        // of temporaries - https://stackoverflow.com/a/48732525/1063961
        match $val {
            tmp => {
                $crate::eprintln!("[{}:{}] {} = {:#?}",
                    file!(), line!(), stringify!($val), &tmp);
                tmp
            }
        }
    };
```

Reply:
I just copied the macro from rust-lang `std` ...

Reply:
Which was probably first written before `$(,)?` was legal. :)

Reply:
> Which was probably first written before `$(,)?` was legal. :)

No, because the next match arm is `($($val:expr),+ $(,)?) => {`

Reply:
anyway, your suggestions seems to work